packages/expo/cli/utils/graphql/client.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  cacheExchange,
+  Client,
+  CombinedError as GraphqlError,
+  createClient as createUrqlClient,
+  dedupExchange,
+  fetchExchange,
+  OperationContext,
+  OperationResult,
+  PromisifiedSource,
+  TypedDocumentNode,
+} from '@urql/core';
+import { retryExchange } from '@urql/exchange-retry';
+import { DocumentNode } from 'graphql';
+import fetch from 'node-fetch';
+
+import * as Log from '../../log';
+import { getExpoApiBaseUrl } from '../api';
+import { getAccessToken, getSessionSecret } from '../user/sessionStorage';
+
+type AccessTokenHeaders = {
+  authorization: string;
+};
+
+type SessionHeaders = {
+  'expo-session': string;
+};
+
+export const graphqlClient = createUrqlClient({
```

Comment:
I know this is orthogonal to this PR but I would like to fix/get rid of that CombinedError message later unless there truly are multiple errors, in which case there is still probably a nicer way to print them than with [CombinedError]